tools/buildtools/ambertools/antechamber.xml

```diff
@@ -0,0 +1,141 @@
+        <option value="resp">RESP</option>
+        <option selected="True" value="bcc">AM1-BCC</option>
+        <option value="cm1">CM1</option>
+        <option value="cm2">CM2</option>
+        <option value="esp">ESP (Kollman)</option>
+        <option value="mul">Mulliken</option>
+        <option value="gas">Gasteiger</option>
+      </param>
+      <param name="at" type="select" label="Atom type" help="Create atom type for the specified forcefield (-at)">
+        <option selected="True" value="gaff">gaff</option>
+        <option value="gaff2">gaff2</option>
+        <option value="amber">amber for PARM94/99/99SB</option>
+        <option value="bcc">bcc</option>
+        <option value="sybyl">sybyl</option>
+      </param>
+      <param name="j" type="select" label="Atom type and bond type prediction index" help="Specify how to run predictions (-j)">
+        <option value="0">0 - No assignment</option>
+        <option value="1">1 - atom type</option>
+        <option value="2">2 - full bond types</option>
+        <option value="3">3 - part bond types</option>
+        <option selected="True" value="4">4 - atom and full bond type (default)</option>
+        <option value="5">5 - atom and part bond type</option>
+      </param>
+    </section>
+    <section name="extraparams" title="Additional Parameters" expanded="false">
+      <param name="pf" type="boolean" checked="true" truevalue="yes" falsevalue="no" label="remove intermediate files (recommended)" help="(-pf)"/>
+      <param name="usenc" type="boolean" checked="true" truevalue="True" falsevalue="False" label="use netcharge (recommended)" help="Can choose to let the charge be inferred by setting this to False."/>
+    </section>
+  </inputs>
+  <outputs>
+    <data format="mol2" name="output1" label="${tool.name}: structure output">
+      <change_format>
+        <when input="selected_output_format" value="pdb" format="pdb"/>
+        <when input="selected_output_format" value="mol2" format="mol2"/>
+        <!-- <when input="selected_format" value="xyz" format="xyz"/> <when input="selected_format" value="tinker" format="tinker"/> <when input="selected_format" value="moldy" format="moldy"/> -->
+      </change_format>
+    </data>
+  </outputs>
+  <tests>
+    <test>
+      <param name="input1" value="LigA.mol2"/>
+      <param name="allparams|nc" value="-2"/>
+      <param name="allparams|c" value="AM1-BCC"/>
+      <param name="allparams|at" value="gaff2"/>
+      <param name="allparams|j" value="5 - atom and part bond type"/>
+      <output name="output1" file="LigA_output.mol2" compare="diff" lines_diff="0">
+        <assert_contents>
+          <has_text text=" 1 C           49.2110    26.9920    85.5530 c2         1 MOL"/>
+        </assert_contents>
+      </output>
+    </test>
+    <test>
+      <param name="input1" value="LigA.pdb"/>
```

Comment:
It seems this file is not being sniffed correctly, try adding `ftype="pdb"`

Reply:
thx was about to ask